src/Sylius/Behat/Page/Admin/Product/CreateSimpleProductPage.php

```diff
@@ -42,20 +44,80 @@ public function specifyPrice($price)
     /**
      * {@inheritdoc}
      */
+    public function addAttribute($attribute, $value)
+    {
+        $this->clickAttributesTabIfItsNotActive();
+
+        $attributeOption = $this->getElement('attributes-choice')->find('css', 'option:contains("' . $attribute . '")');
+        $this->selectElementFromAttributesDropdown($attributeOption->getAttribute('value'));
+
+        $this->getDocument()->pressButton('Add attributes');
+        $this->waitForFormElement();
+
+        $this->getElement('attribute-value', ['%attribute%' => $attribute])->setValue($value);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function removeAttribute($attribute)
+    {
+        $this->clickAttributesTabIfItsNotActive();
+
+        $this->getElement('attribute-delete-button', ['%attribute%' => $attribute])->press();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getRouteName()
+    {
+        return parent::getRouteName() . '_simple';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
     protected function getDefinedElements()
     {
         return array_merge(parent::getDefinedElements(), [
+            'attribute-value' => '.attribute .label:contains("%attribute%") ~ input',
+            'attribute-delete-button' => '.attribute .label:contains("%attribute%") ~ button',
+            'attributes-choice' => 'select[name="sylius_product_attribute_choice"]',
             'code' => '#sylius_product_code',
+            'form' => 'form[name="sylius_product"]',
             'name' => '#sylius_product_translations_en_US_name',
             'price' => '#sylius_product_variant_price',
+            'tab' => '.menu [data-tab="%name%"]',
         ]);
     }
 
     /**
-     * {@inheritdoc}
+     * @param int $id
      */
-    public function getRouteName()
+    private function selectElementFromAttributesDropdown($id)
     {
-        return parent::getRouteName() . '_simple';
+        /** @var Selenium2Driver $driver */
+        $driver = $this->getDriver();
+        Assert::isInstanceOf($driver, Selenium2Driver::class);
+
+        $driver->executeScript('$(\'[name="sylius_product_attribute_choice"]\').dropdown(\'show\');');
+        $driver->executeScript('$(\'[name="sylius_product_attribute_choice"]\').dropdown(\'set selected\', ' . $id . ');');
```

Comment:
``sprintf()`` instead of concatenation WDYT? Btw i think you can avoid this direct script execution. Look at this https://github.com/Sylius/Sylius/pull/3951/files#diff-3cbc2da7c826b61ce38dd6574918b470R78